examples/query_lang/simple_mdoc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "credentials": {
+    "my_mdoc_credential": {
+      "format": "mso_mdoc",
+      "mso_mdoc": {
+        "doctype_values": "org.iso.7367.1.mVR",
+        "alg_values": [ "EdDSA" ],
+        "hash_algorithm_values": [ "SHA-384"]
+      },
+      "claims": [
+        {
+          "namespace": "org.iso.7367.1",
+          "claim_name": "vehicle_holder"
+        },
+        {
+          "namespace": "org.iso.18013.5.1",
+          "claim_name": "first_name"
```

Comment:
```suggestion
          "claim_name": "given_name"
```

Reply:
This was taken from an ISO example, where the claim was called 'first_name'.